packages/@react-types/color/src/index.d.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {BaseSliderProps} from '@react-types/slider';
+import {Color} from '@react-stately/color';
+import {DimensionValue, StyleProps, ValueBase} from '@react-types/shared';
+import {HTMLAttributes, InputHTMLAttributes, RefObject} from 'react';
+
+/** A list of supported color formats. */
+export type ColorFormat = 'hex' | 'hexa' | 'rgb' | 'rgba' | 'hsl' | 'hsla' | 'hsb' | 'hsba';
+
+/** A list of color channels. */
+export type ColorChannel = 'hue' | 'saturation' | 'brightness' | 'lightness' | 'red' | 'green' | 'blue' | 'alpha';
+
+export interface ColorWheelState {
+  readonly value: Color,
+  setValue(value: string | Color): void,
+
+  readonly hue: number,
+  setHue(value: number): void,
+
+  dragging: boolean,
+  setDragging(value: boolean): void
+}
+
+// @ts-ignore
+export interface ColorWheelProps extends BaseSliderProps, ValueBase<Color>, Omit<StyleProps, 'width', 'height'> {
+  onChange?: (value: Color) => void,
+  step?: number,
+  // overriding these to allow passing string:
+  value?: string | Color,
+  defaultValue?: string | Color
+}
+
+export interface ColorWheelAriaProps extends ColorWheelProps {
+  inputRef: RefObject<HTMLElement>,
+  containerRef: RefObject<HTMLElement>,
+  innerRadius: number,
+  outerRadius: number
+}
+
+export interface ColorWheelAriaResult {
```

Comment:
We usually keep the result types for aria hooks in the aria package since it's only used there.